src/main/java/cz/smarteon/loxone/user/User.java

```diff
@@ -0,0 +1,157 @@
+package cz.smarteon.loxone.user;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import cz.smarteon.loxone.LoxoneUuid;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter
+public class User {
+
+    private static final int LOXONE_EPOCH_BEGIN = 1230768000;
```

Comment:
let's introduce new utility class called `LoxoneTime` and extract all these Loxone specific time manipulations from here and from Token.

Can be done subsequent PR: https://github.com/Smarteon/loxone-java/issues/187